tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -163,7 +173,7 @@ def test_more_than_one_match_works_with_different_order(self):
     def test_more_than_one_repo_match(self):
         # XXX: There's a chance that we could infer package names but that is risky
         # repo 1: src/sentry/web/urls.py
-        # repo 2: getsentry/web/urls.py
+        # repo 2: sentry/web/urls.py
```

Comment:
With the potential_match improvements, the previous case would not match anymore, thus, I have change the file.